From 786d5b84d653c019c391fa07e483947156a03e3d Mon Sep 17 00:00:00 2001
From: Andy Grover <agrover@redhat.com>
Date: Sat, 30 Jul 2011 16:02:43 -0700
Subject: [PATCH 1/6] Remove ads from cli welcome msg. Mention help is available.

Signed-off-by: Andy Grover <agrover@redhat.com>
---
 scripts/targetcli |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/targetcli b/scripts/targetcli
index 8af0613..738f02c 100755
--- a/scripts/targetcli
+++ b/scripts/targetcli
@@ -58,9 +58,9 @@ def main():
     shell = TargetCLI('~/.targetcli')
     shell.con.epy_write('''
                         Welcome to the B{targetcli} shell::
-                         Copyright (c) 2011 by RisingTide Systems LLC.
+                         Copyright (c) 2011 by RisingTide Systems LLC.\n
+                         For help on commands, type 'help'.
 
-                        Visit us at U{http://www.risingtidesystems.com}.
                        ''')
     shell.con.display('')
     if not is_root:
-- 
1.7.1

